crates/sui-graphql-rpc/src/context_data/name_service.rs

```diff
@@ -0,0 +1,226 @@
+// Copyright (c) Mysten Labs, Inc.
```

Comment:
Why is this file copied? This code already exists, please use the existing code vs copying it elsewhere.

Reply:
Pulled in sui-json-rpc crate